pkg/steps/artifacts.go

```diff
@@ -651,6 +652,9 @@ func gatherContainerLogsOutput(podClient PodClient, artifactDir, namespace, podN
 	if err != nil {
 		return fmt.Errorf("could not list pod: %v", err)
 	}
+	if len(list.Items) == 0 {
```

Comment:
Why are we doing a list with a field selector instead of a get?

Reply:
That is an excellent question.   Will fix.